composer.json

```diff
@@ -1,17 +1,31 @@
 {
-	"name": "openemr/openemr",
-	"description": "OpenEMR is a Free and Open Source electronic health records and medical practice management application",
-	"license": "GPL",
+    "name": "openemr/openemr",
+    "description": "OpenEMR is a Free and Open Source electronic health records and medical practice management application",
+    "license": "GPL",
     "support": {
         "website": "http://www.open-emr.org/",
         "issues": "https://sourceforge.net/p/openemr/bugs/",
         "forum": "https://sourceforge.net/p/openemr/discussion/",
         "wiki": "http://www.open-emr.org/wiki/index.php/OpenEMR_Wiki_Home_Page",
         "source": "http://github.com/openemr/openemr"
     },
-	"require": {
+    "require": {
 
-        "smarty/smarty": "2.6.29"
-		
-	}
+        "smarty/smarty": "2.6.29",
+        "adodb/adodb-php": "5.20.2"
+        
+    },
+    "require-dev": {
```

Comment:
Basically require-dev is everything you need in development to run your test suite/build, but generally won't need in production. It could also be build tools or such things. There is no huge harm in putting everything into "require" though, it just means you have more code installed on your prod machines, and might slow down your builds a bit depending on how we do them later.